spyql/writer.py

```diff
@@ -18,16 +18,19 @@ def make_writer(writer_name, outputfile, options):
             writer_name = writer_name.upper()
             if writer_name == "CSV":
                 return CSVWriter(outputfile, **options)
-            if writer_name == "JSON":
+            elif writer_name == "JSON":
```

Comment:
again, `elif` is just a matter of style, but if you think it makes it more readable, go ahead